Add support of @RunOnVirtualThread on class for websockets next (#44925)
quarkusio/quarkus: docs/src/main/asciidoc/websockets-next-reference.adoc

Here are the rules governing execution:

* Methods annotated with `@RunOnVirtualThread`, `@Blocking` or `@Transactional` are considered blocking.

> * All Methods in a class annotated with `@RunOnVirtualThread` are considered blocking and run on a virtual thread.

Reviewer: ```suggestion
* Methods declated on a class annotated with `@RunOnVirtualThread` are considered blocking.
```

Reviewer: We should also update the bullet about Kotlin `suspend` functions...

Plus change the last bullet from _"Methods annotated with `@RunOnVirtualThread` must execute on a virtual thread, each invocation spawns a new virtual thread."_ to something like _"Methods annotated with `@RunOnVirtualThread` or declared on a class annotated with `@RunOnVirtualThread` must be executed on a virtual thread, each invocation spawns a new virtual thread."_
